components/TopBarProfileMenu.js

```diff
@@ -38,12 +41,69 @@ const HideGlobalScroll = createGlobalStyle`
 `;
 
 const UserMenuLinkEntry = props => {
+  const { profileMenuLink } = props;
   return (
-    <ListItem mb="6px">
+    <ListItem mb="6px" py={profileMenuLink ? '2' : '0'}>
       <StyledLink as={Link} fontWeight="500" fontSize="14px" lineHeight="20px" color="black.800" {...props} />
     </ListItem>
   );
 };
+UserMenuLinkEntry.propTypes = {
+  profileMenuLink: PropTypes.bool,
+};
+
+const MyAccount = props => {
+  const { LoggedInUser, logout } = props;
+
+  return (
+    <Box as="ul" p={3} my={2}>
+      <Query
+        query={memberInvitationsCountQuery}
+        variables={{ memberCollectiveId: LoggedInUser.CollectiveId }}
+        fetchPolicy="network-only"
+      >
+        {({ data, loading }) =>
+          loading === false && data && data.memberInvitations && data.memberInvitations.length > 0 ? (
+            <UserMenuLinkEntry profileMenuLink href="/member-invitations">
+              <FormattedMessage
+                id="menu.pendingInvitations"
+                defaultMessage="Pending Invitations ({numberOfInvitations})"
+                values={{ numberOfInvitations: data.memberInvitations.length }}
+              />
+            </UserMenuLinkEntry>
+          ) : null
+        }
+      </Query>
+      <UserMenuLinkEntry profileMenuLink href={`/${LoggedInUser.collective.slug}/edit`}>
+        <FormattedMessage id="Settings" defaultMessage="Settings" />
```

Comment:
Minor point, but I think better to stick to a consistent format for `id`s. 🐨 

```suggestion
        <FormattedMessage id="menu.settings" defaultMessage="Settings" />
```

Reply:
This string actually exists already, and it's a good idea to use generic keys for words that are going to be re-used everywhere. Here, for example, we don't want translators to bother translating the word "Settings" for each occurrence it has in the interface (top bar, actions menu, navbar, settings page, etc).

So we should keep this string as-is.